scripts/lib/integration.sh

```diff
@@ -23,6 +23,7 @@ function run_kops_conformance() {
     KOPS_TEST_DURATION=$((SECONDS - START))
     echo "TIMELINE: KOPS tests took $KOPS_TEST_DURATION seconds."
 
+    emit_cloudwatch_metric "kops_test_status" "1"
```

Comment:
Can we also add for bottlerocket, calico and perf tests?

Reply:
Yes, updated it in next commit.